docker/deploy-fedimintd/.env

```diff
@@ -1,18 +1,18 @@
 # Bitcoin auth information
 # generate with https://jlopp.github.io/bitcoin-core-rpc-auth-generator/,
 # default is user: bitcoin, pass: bitcoin
-BITCOIND_RPC_AUTH=bitcoin:54ae356e13a76dc8068e960eb43193cb\$efeeb347a1f0b4a7b7832cc26e68861bc46f89126a8e136ff9932cad47739041
+BITCOIND_RPC_AUTH=bitcoin:54ae356e13a76dc8068e960eb43193cb$efeeb347a1f0b4a7b7832cc26e68861bc46f89126a8e136ff9932cad47739041
```

Comment:
Getting `401 Unauthorized` without this diff. Looks like docker `.env` reads these vars literally, so no need to escape